azure-pipelines.yml

```diff
@@ -259,6 +270,35 @@ jobs:
     artifact: dist-partial-top_earlgrey_nexysvideo
     displayName: 'Upload partial distribution artifacts'
 
+- job: "execute_verilated_tests"
+  displayName: "Execute tests on the Verilated system"
+  pool: "Default"
+  dependsOn:
+    - lint
+    - top_earlgrey_verilator
+    - sw_build
+  condition: eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0')
+  steps:
+  - bash: |
+      sudo apt-get install -y python3 python3-pip build-essential python3-setuptools
+      sudo pip3 install -r python-requirements.txt
+    displayName: 'Install dependencies'
+  - download: current
+    artifact: 'dist-partial-top_earlgrey_verilator'
+    displayName: 'Download verilated simulator'
+  - download: current
+    artifact: 'dist-partial-verilator-sw_build'
+    displayName: 'Download embedded artifacts'
+  - bash: |
+      tar -xvf "$(Pipeline.Workspace)/dist-partial-top_earlgrey_verilator/dist-partial-top_earlgrey_verilator.tar"
+      tar -xvf "$(Pipeline.Workspace)/dist-partial-verilator-sw_build/dist-partial-verilator-sw_build.tar"
+    displayName: 'Unpack pipeline artifacts'
```

Comment:
- Please don't have the pipeline artifacts dropped into the source tree in various places. This makes it rather hard to debug the pipeline and reason about it.
- Can you use the final release artifacts (dist-opentitan) here for testing? Either take the partial distribution artifacts, or even the full distribution artifact created in the 'Create final dist directory out of partial ones' task.